createdisk.sh

```diff
@@ -64,11 +54,10 @@ function resize_and_sparsify {
     fi
 
     guestfish --remote <<EOF
-add-drive $tmpFile
+add-drive $baseDir/$srcFile
 run
 luks-open $partition coreos-root
 mount /dev/mapper/coreos-root /
-xfs-growfs /
```

Comment:
Maybe give slightly more details in the commit log about how this works? The installer provisions a VM with the correct disk size from the start, and on initial boot, RHCOS expands its filesystem to fills the whole disk, so we no longer need to do the resizing ourselves.